doc/tests/block.rst

```diff
@@ -0,0 +1,13 @@
+``block``
+=========
+
+.. versionadded:: 1.23
+    The ``block`` test was added in Twig 1.23.
+
+``block`` checks if a block is defined in the current context of the template.
+
+.. code-block:: jinja
+
+    {% if 'title' is block %}
```

Comment:
@stof Sure. I just don't like the confusing name of the function ``has_block``, that gives the impression: "if the template has a block". That's why I prefer rather the test ``is block``.